nucliadb_protos/nodereader.proto

```diff
@@ -182,8 +182,7 @@ message EntitiesSubgraphRequest {
 }
 
 message EntitiesSubgraphResponse {
-    repeated utils.Relation relations = 1; // Soon deprecated
-    repeated noderesources.IndexRelation new_relations = 2;
+    repeated noderesources.IndexRelation relations = 2;
```

Comment:
Reserve the removed field index to avoid future accidental usage:
```suggestion
    reserved 1;
    repeated noderesources.IndexRelation relations = 2;
```